### PR TITLE
Update Calendar.m

### PR DESCRIPTION
**Pull request by pablosomarriba**
Fix ios 15.1 -- Modifications to the layout engine must not be performed from a background thread after it has been accessed from the main thread.

### CODE REVIEW

**Review by pendelru — APPROVED** [review]
Works for us.

**Comment by marcotipitto** [comment]
Could we get this merged?

**Comment by pmcquay** [comment]
we have seen this issue as well, and are waiting on this PR to get merged.

**Comment by btate** [comment]
@pendelru Do you have a timeline on merging this in?

**Comment by pendelru** [comment]
@btate I'd like to see this PR merged too, but only those with write access to this repo can merge PRs. I don't have write access.

**Comment by btate** [comment]
@pendelru haha. Sorry about that. For some reason I thought you were in charge.

**Comment by btate** [comment]
@EddyVerbruggen Are you able to accommodate us?

**Comment by bhandaribhumin** [comment]
@EddyVerbruggen I am waiting, who will merge?  We are using CI build so I am looking to see when the pull request will be merged. Because of this our deployment has stopped.

**Comment by bbialas** [comment]
+1 If the fix is confirmed, please merge it ASAP. Thank you.

**Comment by B4RteQPl** [comment]
@EddyVerbruggen could you merge it, please?

**Review by naqeeb-klabs — APPROVED** [review]

**Comment by phyr0s** [comment]
@EddyVerbruggen  merge please works fine

**Pull request merged**
